src/App.vue

```diff
@@ -269,7 +272,12 @@ footer {
 .chats .md-list {
     max-width: 90%;
 }
-
+a:not(.md-button) {
+  color: #2e7eed!important;
```

Comment:
can we make more specified selector, add additional class, instead of using important.